public_html/includes/classes/forms.php

```diff
@@ -1137,7 +1137,15 @@ public static function buildEditTable($formID) {
 						);
 
 				foreach ($form['fields'] as $field) {
-					$temp[] = sprintf('<input type="%s" style="%s" name="%s_%s" value="%s" readonly />',
+          // checking if type is text then we will use textarea instead of an input
+					if ($field['type'] === 'text') {
```

Comment:
This change will change all fields of type "text" to a text area. This isn't the desired result, because there are many cases where we want a single line text input and not a text area. For example, there is no reason to have a text area for a metadata schema type which is only 2 or 3 characters long. 

The correct way to do what we discussed last thursday is to update the form creater so that Metadata forms can accept multiline text areas. Then  this section needs expanded to handle different typed. Textarea and select are the two with immediate needs. 

Once MFCS is updated to support multiline text inputs in the form creator and in the edit table, on metadata forms, PEC needs updated. Drag over a multiline text on the metadata heading forms. Name everything the same, get rid of the original. That should effectively change the type.